OxfordBodleian/Juel-Jensen/BDLaethg31.xml

```diff
@@ -255,32 +255,35 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                     </desc>
                                     <q xml:lang="gez">ወልደ፡ ጊዮርጊስ፡ ምንሱቅ<supplied reason="omitted">፡</supplied> ገዛኡት፡ በኑሰ<supplied reason="omitted">፡</supplied> ዘለጠአ።</q>
                                 </item>
+                                
                                 <item xml:id="e1">
-                                    <desc>A white sticker is glued to the inner side of the front board. The name of <persName ref="PRS5782JuelJen"/> is printed on it
-                                        (<foreign xml:lang="gez">ቤንት፡ ዩውል፡ የንሰን።</foreign>) and the signature "MS. Aeth. g. 31" is written in pencil below it.
+                                    <desc>
+                                        Omissions are sometimes written interlineally (e.g. on <locus target="#1r #8r #24v"/>). Erasures on e.g. <locus target="#2v #3r #41v"/>.
```

Comment:
I think it doesn't make sense to say that "Omission are… written…". I checked the folios quoted and I think we could formulate it: "Erroneously omitted words and letters have occasionally been added interlineally…"

Reply:
@MassimoVilla